metron-platform/metron-pcap-backend/src/test/java/org/apache/metron/pcap/integration/PcapTopologyIntegrationTest.java

```diff
@@ -90,23 +90,6 @@ public boolean accept(File dir, String name) {
   }
 
```

Comment:
I think until this is completely deprecated, we should keep the test, but disable it.